src/jira.py

```diff
@@ -0,0 +1,144 @@
+import base64
+import re
+import requests
+import pprint
+import time
+
+from src.access import get_access_params
+from src.board import Issue
+
+
+class JiraBoard:
+
+    def __init__(self, project_key):
+        """
+        Create a Project storing all issues belonging to the provided project key
+
+        :param str project_key: The Jira project key e.g. "TEST"
+        """
+        self.url = get_access_params('jira')['options']['server']
+        self.token = get_access_params('jira')['api_token']
```

Comment:
Why do we not need base64 encoding here?

Reply:
fixed